src/routes/transactions/entities/transaction.entity.ts

```diff
@@ -39,6 +40,7 @@ export class Transaction {
       { $ref: getSchemaPath(CustomTransactionInfo) },
       { $ref: getSchemaPath(SettingsChangeTransaction) },
       { $ref: getSchemaPath(SwapOrderTransactionInfo) },
+      { $ref: getSchemaPath(TwapOrderTransactionInfo) },
```

Comment:
`TwapOrderTransactionInfo` needs to be included in the `@ApiExtraModels` as well in order to be available within the class.

```
@ApiExtraModels(
  CreationTransactionInfo,
  CustomTransactionInfo,
  SettingsChangeTransaction,
  TransferTransactionInfo,
  ModuleExecutionInfo,
  MultisigExecutionInfo,
  SwapOrderTransactionInfo,
  TwapOrderTransactionInfo,
)
```

Reply:
oh, thanks! Missed that.